server/src/db/migrations/20220503184204_createReviews.cjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("reviews", (table) => {
+        table.bigIncrements("id")
+        table.string("title").notNullable()
+        table.integer("rating").notNullable()
+        table.string("content")
```

Comment:
might consider making this a type of `text` for a larger character limit on the content of the review